src/List/List.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Children } from 'react';
```

Comment:
What's this?

Reply:
no clue lol, probably copied over from MDX where there was no highlighting